### DISCUSSION TITLE
Spaces in file name changes to dots while uploading via releases section

### DISCUSSION THREAD

**Discussion by kuldeep-1k** [community/community#137014] [Other Feature Feedback, Questions, & Ideas]
Question

### Body

When I'm trying to attach a file having name with spaces like 'My Package.pkg', it changes to 'My.Package.pkg' in the releases section while creating a release. I don't want that to happen. How to achieve this?

**Comment by phuyalgaurav** [comment]
It's a default git behavior, consider adding underscore or a hyphen (my-package.pkg, my_package.pkg). If you really want to circumvent this, you can upload it as a .zip. If you upload the package using github's interface (From the site itself, i think ) it does not add the "." instead of spaces.

**Reply by ebndev** [reply]
Hi @PhuyalGaurav, thanks for taking the time to answer!

@kuldeep-1k if the response above helped, please feel encouraged to mark it as the answer. 😄

**Comment by joemar25** [comment]
Hi @kuldeep-1k ,

It looks like @PhuyalGaurav provided a great workaround by using underscores or hyphens in the file name. Another approach you might consider, if you want to keep the spaces in the filename but avoid them turning into dots, is to compress the file (e.g., into a .zip or .tar.gz archive) before uploading it. When files are compressed into an archive, the original file names inside the archive remain intact, spaces included.

Additionally, you could use a script or command to rename the files before creating the release if you're automating this process. For example, you could use a simple bash script to replace spaces with underscores or hyphens automatically before uploading.

Hope this helps as another option!
